package.json

```diff
@@ -193,11 +193,12 @@
                     "absolute"
                 ],
                 "pattern": {
-                    "regexp": "(?:Autobuild::CMake|Autobuild::Orogen):(?:[^:]+):build: (.+):(\\d+):(\\d+):\\s+error:\\s+(.+)",
+                    "regexp": "(?:Autobuild::CMake|Autobuild::Orogen):(?:[^:]+):build: (.+):(\\d+):(\\d+):\\s+((?:fatal\\s+)?error):\\s+(.+)",
```

Comment:
I don't think there is a "fatal error" severity.

Reply:
Good catch. It the non-matching group was not supposed to be nested... Updated.